core/views.py

```diff
@@ -117,7 +117,30 @@ class ReportsView(LoginRequiredMixin, TemplateView):
 
 @login_required
 def entries_csv_export(request):
-    dataset = EntryResource().export()
-    response = HttpResponse(dataset.csv, content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename="entries.csv"'
+    client = request.GET.get('project_client')
```

Comment:
This is the long way to go about this, we already have a dict of everything in GET, you can just pass that directly to the filter using **request.GET , you need to probably split it out first, verify all the parameters are valid and remove the export_format first though.